src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1001,6 +976,10 @@ class CreateVmModal extends React.Component {
         const { nodeMaxMemory, nodeDevices, networks, osInfoList, loggedUser, storagePools, vms } = this.props;
         const validationFailed = this.state.validate && validateParams({ ...this.state, osInfoList, nodeMaxMemory, vms: vms.filter(vm => vm.connectionName == this.state.connectionName) });
 
+        const unattendedInstructionsMessage = _("Enter root and/or user information to enable unattended installation.");
+        const unattendedUnavailableMessage = _("Automated installs are only available when downloading an image or using cloud-init.");
+        const unattendedOsUnsupportedMessageFormat = os => _(`${os} does not support unattended installation.`);
```

Comment:
This i18n pattern does not work. *Never* use _() with backtick-formatting, as it's not translatable. You *first* need to translate the template and *then* fill it in with values. Please use cockpit.format() (of course not here in the constant, but where you use it)

Reply:
I filed PR #655 to fix and spot that.